.eslintignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+coverage
+docs
+modules
+public
+ui
+sponsor-dapp-v2
```

Comment:
for now we cant run eslint in the dapp directories. this requires a number of react specific eslint plugins which I have not set up yet. Prettier still runs in these directories though.